playground/Testcases.tsx

```diff
@@ -0,0 +1,114 @@
+import { useRef, useEffect, Fragment } from "react";
+
+import { FLOWCHART_DIAGRAM_TESTCASES } from "./testcases/flowchart";
+import { SEQUENCE_DIAGRAM_TESTCASES } from "./testcases/sequence.ts";
+import { CLASS_DIAGRAM_TESTCASES } from "./testcases/class.ts";
+import { UNSUPPORTED_DIAGRAM_TESTCASES } from "./testcases/unsupported.ts";
+
+import type { UpdateMermaidDefinition } from "./index.tsx";
+import { MermaidDiagram } from "./MermaidDiagram.tsx";
+
+interface TestCaseProps {
+  name: string;
+  testcases: { name: string; definition: string }[];
+  onChange: (definition: string, activeTestcaseIndex: number) => void;
+  error: string | null;
+  activeTestcase?: number;
+}
+
+const Testcase = ({
```

Comment:
Lets move this component to a file `SingleTestCase.tsx`

Reply:
done in [c0752ad](https://github.com/excalidraw/mermaid-to-excalidraw/pull/52/commits/c0752adcd61e6695a397151c532430dc0c03a36e)